src/code/z_player_lib.c

```diff
@@ -599,11 +614,17 @@ u8 sEyeMouthIndexes[][2] = {
     { 7, 2 }, { 0, 2 }, { 3, 0 }, { 4, 0 }, { 2, 2 }, { 1, 1 }, { 0, 2 }, { 0, 0 },
 };
 
-u8* sEyeTextures[] = {
+/*
+ * The dev who set these up likely knew that these textures would be in the same spot for child and adult link.
+ * Though this works its a problem for mods.
+ * They will be left as addresses until object_link_boy is completed where they will
+ * then have the symbols from adult links object.
+ */
```

Comment:
would try to refrain from mentioning the devs directly. I recommend something like 
```
Link's eye and mouth textures are placed at the exact same place in adult and child Link's respective object files. This allows the array to only contain the symbols for one file and have it apply to both.
This is a problem for shiftability, and changes will need to be made in the code to account for this in a modding scenario.

The symbols from adult Link's object are used here.
```